rails/rails_releases/rails_benchmarks/runner

```diff
@@ -11,18 +11,21 @@ while ! exec 6<>/dev/tcp/${MYSQL_PORT_3306_TCP_ADDR}/${MYSQL_PORT_3306_TCP_PORT}
   sleep 1
 done
 
+
+dropdb --if-exists -h "$POSTGRES_PORT_5432_TCP_ADDR" -p "$POSTGRES_PORT_5432_TCP_PORT" -U postgres rubybench
 createdb -h "$POSTGRES_PORT_5432_TCP_ADDR" -p "$POSTGRES_PORT_5432_TCP_PORT" -U postgres rubybench
+echo "DROP DATABASE IF EXISTS rubybench" | mysql -u root -h "$MYSQL_PORT_3306_TCP_ADDR" -P "MYSQL_PORT_3306_TCP_PORT"
```

Comment:
I've been running it locally and I don't create new containers for the databases every time. I wrote a script to run the the benchmarks for the different releases of Rails. It's not necessary if you create new containers.